### PR TITLE
feature/ads-750: automate tokens docs

### PR DESCRIPTION
**Pull request by APJohns**

### CODE REVIEW

File: .storybook/preview.ts [hunk 1]
```diff
@@ -16,7 +16,8 @@ const preview: Preview = {
         method: 'alphabetical'
         // TODO: uncomment when we build out these sections
         // order: ['Getting started', 'Tokens', 'Components', 'Patterns']
```
Comment (kaseyllabelle) [inline .storybook/preview.ts]: tokens feel a little lost at the bottom of the panel. thoughts on uncommenting this now?

File: .storybook/storybook.scss [hunk 2]
```diff
@@ -18,3 +23,51 @@ body {
     fill: var(--bsc-logo-color-secondary);
   }
 }
+
+table {
```
Comment (kaseyllabelle) [inline .storybook/storybook.scss]: thoughts on scoping this to a class? maybe .docs-table or something?

File: src/stories/tokens/TokensTable.tsx [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+
+  useEffect(() => {
+    getTokens();
+  }, [getTokens]);
+
+  return (
+    <table>
+      <thead>
+        <tr>
+          <th>Name</th>
```
Comment (kaseyllabelle) [inline src/stories/tokens/TokensTable.tsx:35]: thoughts on token vs name? and maybe splitting into columns and formatting as css custom properties, scss, and less vars to mimic what's on the docs site?

Comment (APJohns) [reply src/stories/tokens/TokensTable.tsx:35]: The names of the variables are all the same, all that would change would be if it starts with --, $, or @. Then we also have the json version too (also same names).

**Review by kaseyllabelle — APPROVED** [review]

**Pull request merged**